### PR TITLE
Add Text.json.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: api/Text.json [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+            "safari_ios": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```
Comment (chrisdavidmills) [inline api/Text.json]: Looks like this has been removed from the spec, so this would be false

File: api/Text.json [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Text/wholeText",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
```
Comment (chrisdavidmills) [inline api/Text.json]: 3.5

File: api/Text.json [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+            },
+            "firefox_android": {
+              "version_added": true
```
Comment (chrisdavidmills) [inline api/Text.json]: 3.5

File: api/Text.json [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "assignedSlot": {
```
Comment (chrisdavidmills) [inline api/Text.json]: This shouldn't be in here. Copy paste error?

File: api/Text.json [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+            "opera": {
+              "version_added": true,
+              "version_removed": "32"
+            },
+            "opera_android": {
+              "version_added": true,
+              "version_removed": "32"
+            },
+            "safari": {
+              "version_added": true
```
Comment (chrisdavidmills) [inline api/Text.json]: I've tested this in Safari, and it is no longer supported. The same is probably true for iOS. Should we just put false for now? That would be less misleading than saying it is true.

File: api/Text.json [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+              "version_added": "1",
+              "notes": [
+                "Before Chrome 30, the <code>offset</code> argument was optional."
+              ]
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
```
Comment (chrisdavidmills) [inline api/Text.json]: 1

File: api/Text.json [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
+            },
+            "firefox_android": {
+              "version_added": true
```
Comment (chrisdavidmills) [inline api/Text.json]: 4

File: api/Text.json (hunk 4 above)
Comment (jpmedley) [reply api/Text.json]: The error was the path in `mdn_url`.

**Comment by jpmedley** [comment]
PTAL

File: api/Text.json (hunk 3 above)
Comment (jpmedley) [reply api/Text.json]: @chrisdavidmills Travis doesn't like 3.5 as a number for Firefox.

Comment (chrisdavidmills) [reply api/Text.json]: OK, just put 4 for now.

File: api/Text.json (hunk 4 above)
Comment (chrisdavidmills) [reply api/Text.json]: OK

File: api/Text.json (hunk 1 above)
Comment (chrisdavidmills) [reply api/Text.json]: Still needs fixing?

**Comment by jpmedley** [comment]
After all this time, I'm still a bit hazy on how you classify some things. This API is part of a whatwg living standard. That would make it **not** on the standard track by MDN's definition?

**Comment by chrisdavidmills** [comment]
I'm sorry for the confusion - I was referring to replaceWholeText(), which you've marked as standards track true, but it's no longer in the WHATWG spec. I put the comment in the wrong place for some reason.

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**